authorization/service/deleteDevices_test.go

```diff
@@ -6,10 +6,14 @@ import (
 	"testing"
 
 	"github.com/plgd-dev/cloud/authorization/pb"
+	"github.com/plgd-dev/cloud/pkg/net/grpc"
 	kitNetGrpc "github.com/plgd-dev/cloud/pkg/net/grpc"
 	"github.com/stretchr/testify/require"
 )
 
+const jwtWithSubTestUser = `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ0ZXN0VXNlcklEIn0.6EZJidMCJ5UMwyttpwUNer-GdsBmPH1_ckH8ZU-SRpo`
```

Comment:
This works, though it's not obvious what does the token contain, so the programmer has to do an additional step and decode it if he wants to see. I'm thinking we could create a encoding function that takes map of claims and returns the encoded token as string - see https://pkg.go.dev/github.com/golang-jwt/jwt#example-New-Hmac

Reply:
Thx - fixed